src/components/OnboardingDialog/index.js

```diff
@@ -0,0 +1,156 @@
+import React, { useState } from 'react'
+import styled from '@emotion/styled'
+import PropTypes from 'prop-types'
+import {
+  radius,
+  shadow,
+  space,
+  duration,
+  transition,
+  color,
+  device,
+} from '../../theme'
+import { useTransition } from '../../hooks'
+import { EXITED, ENTERING, ENTERED, EXITING } from '../../hooks/useTransition'
+
+const BodyWrapper = styled.div`
+  background-color: white;
```

Comment:
`color.nova` (maybe good to do this for everything is a separate commit)

Reply:
I would rather do that in another PR.